src/MissionCreator.sol

```diff
@@ -98,15 +103,14 @@ contract MissionCreator is Ownable, IERC721Receiver {
                 ownerMustSendPayouts: false, // Anyone can send this project's payouts to the splits specified in the `splitGroups` property below.
                 holdFees: false, // Fees are not held.
                 useTotalSurplusForCashOuts: false, // Cash outs are made from each terminal independently.
-                useDataHookForPay: false, // The project does not use a data hook for payouts.
+                useDataHookForPay: true, // The project does not use a data hook for payouts.
```

Comment:
True = we don't use a data hook? Or is the comment here wrong?

Reply:
bad comment